google/oauth2/credentials.py

```diff
@@ -335,10 +351,11 @@ def refresh(self, request):
 
         if scopes and "scope" in grant_response:
             requested_scopes = frozenset(scopes)
-            granted_scopes = frozenset(grant_response["scope"].split())
+            self._granted_scopes = grant_response["scope"].split()
+            granted_scopes = frozenset(self._granted_scopes)
             scopes_requested_but_not_granted = requested_scopes - granted_scopes
             if scopes_requested_but_not_granted:
-                raise exceptions.RefreshError(
+                _LOGGER.warning(
```

Comment:
Can you give some extra context on why changing the behavior is OK?

Seems reasonable, just want to understand the change.